### PR TITLE
Fix error expected a value of type 'RTCVideoRendererWeb'

### PR DESCRIPTION
**Pull request by samuelg123**
This will fix issue #365

### CODE REVIEW

File: test/unit/web/rtc_videw_view_test.dart [hunk 1]
```diff
@@ -1,12 +1,12 @@
 @TestOn('browser')
 import 'package:flutter_test/flutter_test.dart';
-import 'package:flutter_webrtc/flutter_webrtc.dart';
+import 'package:flutter_webrtc/flutter_webrtc.dart' hide RTCVideoRenderer;
```
Comment (cloudwebrtc) [inline test/unit/web/rtc_videw_view_test.dart:3]: I don’t quite understand why `hide RTCVideoRenderer` here

**Comment by cloudwebrtc** [comment]
@samuelg123 fixed https://github.com/flutter-webrtc/flutter-webrtc/commit/975a9f00fe37e7e32d6b3ecfc23b9f47e340126c

**Comment by samuelg123** [comment]
Great. I will close this.

File: test/unit/web/rtc_videw_view_test.dart (hunk 1 above)
Comment (wer-mathurin) [reply test/unit/web/rtc_videw_view_test.dart:3]: This an error.  But this test is not working anyway.  I will write a working test when the below issue will be resolve.
The annotation @TestOn is not working.  see https://github.com/flutter/flutter/issues/66045

**Pull request closed without merging**